### PR TITLE
chore(cvsb-18934): update deps and scripts

### PR DESCRIPTION
**Pull request by JonLazariniDD**

### CODE REVIEW

**Review by bristol-mike-jones — COMMENTED** [review]
Can we add dependabot & PR template ?

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "security-checks": "git secrets --scan && git log -p | scanrepo",
     "sonar-scanner": "BRANCH=local npm run tslint && npm run coverage && sonar-scanner",
     "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --coverage",
```
Comment (m-ujaffer) [inline package.json:21]: need to update to only call unit tests.

Comment (JonLazariniDD) [reply package.json:21]: done feel free to resolve

**Comment by JonLazariniDD** [comment]
added the config, i didn't realise it was empty, thanks for spotting

**Pull request merged**